packages/yoastseo/src/scoring/assessments/inclusiveLanguage/configuration/disabilityAssessments.js

```diff
@@ -17,6 +21,8 @@ const potentiallyHarmfulTwoAlternatives = "Avoid using <i>%1$s</i> as it is pote
 
 const learnMoreUrl = "https://yoa.st/inclusive-language-disability";
 
+const punctuationList = punctuationRegexString.split( "" );
```

Comment:
The fact that I use this in every inclusive language config file is pretty violating towards DRY. But I did not find it worth to put this in a separate file.